README.md

```diff
@@ -0,0 +1,89 @@
+# Yocto Build Environment as Docker Container
+
+This repository aims to provide a complete yocto build environment for building
+BISDN Linux in a docker container.
+
+## Build Environment
+
+The build environment is based on the recommendations documented in the yocto
+project for Ubuntu and Debian
+[here](https://docs.yoctoproject.org/3.1.7/ref-manual/ref-system-requirements.html#ubuntu-and-debian).
+In addition to the packages mentioned there, it also includes
+[libelf](https://directory.fsf.org/wiki/Libelf) with development headers and
+[PyYAML](https://pyyaml.org), since those are needed to build some of the
+packages included in BISDN Linux.
+Last but not least, the environment also includes
+['repo'](https://android.googlesource.com/tools/repo), a tool developed by google
+to manage multiple git repositories and their respective versions from one
+manifest file. Since 'repo' requires python and building Open Network Linux
+depends on python2, the environment includes both python3, as well as python2.
+
+In addition to providing all packages and dependencies needed for building BISDN
+Linux with the yocto build chain, the environment already includes a
+minimalistic [.gitconfig](./gitconfig) needed when using the aforementioned
+'repo' tool and sets the locale to 'en_US.UTF-8'. Since the yocto build tool
+'bitbake' does not allow building as 'root' user, the environment includes a
+'builder' user, which is used by default when running the '-ci' version of the
```

Comment:
```suggestion
'builder' user, which is used by default when running the 'ci' version of the
```
Since we mention 'base' and 'ci' targets below, should we drop the dash (-) here?

Reply:
done